dbms/src/Storages/DeltaMerge/SkippableBlockInputStream.h

```diff
@@ -217,6 +218,10 @@ class ConcatSkippableBlockInputStream : public SkippableBlockInputStream
     {
         if (likely(scan_context != nullptr))
         {
+            LocalAdmissionController::global_instance->consumeResource(
+                scan_context->resource_group_name,
+                bytesToRU(bytes),
+                0);
```

Comment:
what if disable resource control?

Reply:
All public member function of `LAC` will be dummy if `LAC` is not start